web3/utils/module_testing/eth_module.py

```diff
@@ -159,6 +159,11 @@ def test_eth_getCode_invalid_address(self, web3, math_contract):
         with pytest.raises(InvalidAddress):
             web3.eth.getCode(math_contract.address.lower())
 
+    def test_eth_getCode_with_block_identifier(self, web3, emitter_contract):
+        code = web3.eth.getCode(emitter_contract.address, block_identifier=web3.eth.blockNumber)
```

Comment:
> `block_identifier=web3.eth.blockNumber`

Don't know if this is the right thing to do. The ideal way would be to deploy a contract and get the `blockNumber` of the transaction in which it was mined and then call this method. 
However, I inspected the tests and found that only block was added. I wanted to fix this quick, so didn't spend much time. Let me know what guys think about it!

Reply:
This is an acceptable solution for now.  :+1: